vmdb/db/migrate/20150401223517_container_nodes_hardware.rb

```diff
@@ -0,0 +1,8 @@
+class ContainerNodesHardware < ActiveRecord::Migration
+  def change
+    add_column :hardwares, :container_node_id, :bigint
+    add_column :container_nodes, :infra_machine_id, :string
+    add_column :container_nodes, :machine_id, :string
+    add_column :container_nodes, :system_uuid, :string
```

Comment:
From what I can tell from the various refreshers, we may not be collecting this information from any of the providers.

It's possible that we pick up the bios system uuid for [VMWare vms](https://github.com/ManageIQ/manageiq/blob/d9b3679d8d4cd1e30a95b362c5e88ed59aaa39f7/vmdb/app/models/ems_refresh/parsers/vc.rb#L724).

Other than that, I don't see any strong correlation for the other providers and the other attributes you have.